feature/com-264

amnad001 opened the pull request:
made a page for  Project Standard for Single Sign-On (SSO)

Commit:
feature/com-264

SireeKoolenWijkstra reviewed (changes requested):
Can you add that the documentation is only visible to developers with access to the organisation Amsterdam in Github? But otherwise, I like it. :+1::skin-tone-4:

Commit:
Change based on the PR

Noted that the documentation is only visible to developers of Gemeente Amsterdam

SireeKoolenWijkstra reviewed (approved):
LGTM
